build/lib/release.sh

```diff
@@ -214,10 +214,40 @@ function kube::release::package_node_tarballs() {
   done
 }
 
+function kube::release::is_current_builder_missing_target_arch() {
+  local current_builder
+  current_builder="$(docker buildx inspect)"
+  if grep -q "^Driver: docker$"  <<<"${current_builder}"; then
+    return 0
+  fi
+  local platform
+  for platform in "${KUBE_SERVER_PLATFORMS[@]}"; do
+    if ! grep -q "${platform}" <<<"${current_builder}"; then
+      return 0
+    fi
+  done
+  return 1
+}
+
 # Package up all of the server binaries in docker images
 function kube::release::build_server_images() {
   # Clean out any old images
   rm -rf "${RELEASE_IMAGES}"
+
+  if kube::release::is_current_builder_missing_target_arch; then
+    export DOCKER_CLI_EXPERIMENTAL=enabled
+    # We probably want to update this in the future to run:
+    # ${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset --persistent yes
+    # but to do that we will have to update the kube::build::verify_prereqs to make sure qemu-user-static is installed
+    # on the host and will need to update the CI images to have that pre-installed.
+    docker run --rm --privileged multiarch/qemu-user-static:5.2.0-2 --reset -p yes
```

Comment:
we're *not* OK with `docker run --privileged foo:tag` in this repo

also in retrospect: the build image is not one I'd expect to be multi-arch anyhow?
for the other images we'll need this, but currently the pattern is indeed that script instead https://cs.k8s.io/?q=third_party%2Fmultiarch%2F&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

Reply:
they also don't have to be installed on the host https://github.com/kubernetes/kubernetes/blob/186f934e4ccb23e7dca03cc836f11f89e7d99e08/test/images/image-util.sh#L169

Reply:
1. Yup we don't need multi-arch for build-image. Updated the PR description.
2. When using 
${sudo} "${KUBE_ROOT}/third_party/multiarch/qemu-user-static/register/register.sh" --reset --persistent yes
and then creating a builder using 
only creates a builder for: linux/amd64, linux/386
3. Why is downloading a tar safe (https://github.com/multiarch/qemu-user-static/releases/download/"${QEMUVERSION}"/x86_64_qemu-"${QEMUARCHS[$arch]}"-static.tar.gz)?

Reply:
Oh I didn't know that they don't have to be installed on the host. I reverted it to use `third_party/multiarch/qemu-user-static/register/register.sh" --reset --persistent yes`

Reply:
Discussed offline, Ben found a way to unblock kube-apiserver without the need for qemu: https://github.com/kubernetes/kubernetes/pull/96134#discussion_r580704333